templates/query.html

```diff
@@ -10,11 +10,12 @@
 {% block body_class %}query db-{{ database|to_css_class }}{% endblock %}
 
 {% block content %}
-
 {% if canned_query %}
 <h2 class="title">{{ canned_query.replace("-", " ") }}</h2>
 {% endif %}
-
+<div class="center">
```

Comment:
Total nitpick: How about `<p>` for this?

Reply:
Ha, I'm open to it, but that's not a fix that seems to work for me on my local repo...

Reply:
If `<p>` creates weird padding issues just forget about it.